3p/integration.js

```diff
@@ -412,6 +414,7 @@ const defaultAllowedTypesInCustomFrame = [
   'doubleclick',
   'yieldbot',
   '_ping_',
+  'medyanet',
```

Comment:
this list exists for legacy reason. you don't need to change it

Reply:
I removed this line.